var/spack/repos/builtin/packages/ds/package.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Ds(AutotoolsPackage):
+    """SAOImage DS9 is an astronomical imaging and data visualization
+       application."""
+
+    homepage = "http://ds9.si.edu/"
+    url      = "http://ds9.si.edu/download/source/ds9.8.0rc6.tar.gz"
+
+    version('9.8.0rc6', sha256='9c926dbe5475529b4de456058605e61e523f442993a3aa188f5d3fbd59a9af7a')
```

Comment:
I typically don't add release candidates as the default version, only stable releases. Can you add the most recent stable release? If you still want to be able to install 9.8.0rc6, you can mark the most recent stable release with `preferred=True` and it will be installed by default.

Reply:
Recipe changed to point to 9.8.0 stable release